docs/quickstart/quickstart_chains/cosmos.md

```diff
@@ -10,12 +10,17 @@ Before we begin, make sure that you have initialised your project using the prov
 
 Note that we are using Juno as the example here, but SubQuery supports all the following Cosmos networks and you can quickly initialise a new project in any of them using the `subql init` command:
 
-- Juno
-- CosmosHub
-- Fetch.ai
-- Stargaze
+- [CosmosHub](https://github.com/subquery/cosmos-subql-starter/tree/main/Cosmos/cosmos-starter)
+- [Fetch.ai](https://github.com/subquery/cosmos-subql-starter/tree/main/Fetch.ai/fetchhub-starter)
+- [Stargaze](https://github.com/subquery/cosmos-subql-starter/tree/main/Stargaze/stargaze-starter)
 - Osmosis
```

Comment:
@jamesbayly not sure where to find starter project for this Osmosis? 

![Screen Shot 2022-10-18 at 11 30 14 AM](https://user-images.githubusercontent.com/24416574/196295768-9cdd4182-f172-40bf-87e3-f20f6b4ed993.png)